examples/0-counter-part.elm

```diff
@@ -47,7 +45,7 @@ update msg model =
       init
 
     CounterMsg msg' -> 
-      Parts.update CounterMsg msg' model
+      Counters.pass CounterMsg msg' model
```

Comment:
Isn't this already the case in elm-mdl if you don't want all components? http://package.elm-lang.org/packages/debois/elm-mdl/6.1.0/Material#optimising-for-size

I'll add a example showing, how to "unify" these.